api/src/env.ts

```diff
@@ -291,6 +293,9 @@ const typeMap: Record<string, string> = {
 	DB_PORT: 'number',
 
 	DB_EXCLUDE_TABLES: 'array',
+
+	CACHE_SKIP_ALLOWED: 'boolean',
+
```

Comment:
Do we need this? I guess there are quite a lot of other boolean envs where we don't force this.

Reply:
It is definitely not necessary for this PR to work. This is more of an attempt at "continuing" the discussion in https://github.com/directus/directus/pull/17309#discussion_r1090772083 to start doing it more, especially for new env vars 😄 we can also opt out of this for now depending on how we'd want to tackle this env-type-map topic 👍

Reply:
I see, in that case let's keep it like this 😃👍